reactioncommerce/reaction-component-library
# (feat): 64 Checkbox Component

Resolves #64  
Impact: **major**  
Type: **feature**





## Component
- Add a CheckboxInput component based on these [specs](https://zpl.io/bzYjeKA).
- Accessibility: I added a light blue fuzzy border for the `:focus`
- Snyk: I'll rebase master as soon as snyk passes on master.

## Screenshots
Checked, unchecked, disabled
<img width="643" alt="screen shot 2018-07-17 at 4 11 06 pm" src="https://user-images.githubusercontent.com/3673236/42850354-0bf18118-89dc-11e8-91f8-8407b5f30e60.png">

Focus
<img width="787" alt="screen shot 2018-07-17 at 4 20 57 pm" src="https://user-images.githubusercontent.com/3673236/42850648-776f737c-89dd-11e8-888a-6aa1ee20bbf8.png">

## URL
https://deploy-preview-78--stoic-hodgkin-c0179e.netlify.com/#!/Checkbox

## Testing
- Go to https://deploy-preview-78--stoic-hodgkin-c0179e.netlify.com/#!/Checkbox
- Use mouse to check and uncheck checkboxes
- Make sure disabled checkbox cannot be checked
- Use keyboard to check and uncheck checkboxes
